src/components/Card.tsx

```diff
@@ -0,0 +1,133 @@
+import { formatTime } from "./FormattTime";
+import "../css/Card.css";
+import { useRef, useEffect, useState } from "react";
+import { MODALS_ID } from "../modal/modalcomponents/ModalId";
+import AlertModal from "../modal/AlertModal";
+import AddLinkModal from "../modal/AddLinkModal";
+import kebob from "../image/kebab.png";
+import starImage from "../image/star.png";
+
+function formatDate(value: number) {
+  const date = new Date(value);
+  return `${date.getFullYear()}. ${date.getMonth() + 1}. ${date.getDate()}`;
+}
+interface Item {
+  url: string;
+  imageSource: string;
+  image_source: string;
+  title: string;
+  createdAt: number;
+  created_at: number;
+  description: string;
+}
+
+interface CardProps {
+  item: Item;
+  folderId: string;
+  userFolder: { data: { id: number; name: string; link: { count: number } }[] };
```

Comment:
P2: 속성명으로는 배열인지 알 수 없네요. 어떤 값인지 알 수 있는 이름이면 좋겠습니다.

```suggestion
  userFolders: { data: { id: number; name: string; link: { count: number } }[] };
```